src/components/login/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { PropTypes, Component } from 'react'
+import { translate } from 'react-i18next'
+
+class Login extends Component {
+	constructor (props) {
+		super(props)
+		this._handleSubmit = this.handleSubmit.bind(this)
```

Comment:
both proposed solutions only need to use "handleSubmit" instead of "_handleSubmit" in your renderer